testing/python/kernel/test_tilelang_kernel_bf16_gemm_tcgen5_ts.py

```
@@ -0,0 +1,185 @@
+            S_local = T.alloc_fragment((bM, bN1), accum_dtype)
+            P_local = T.alloc_fragment((bM, bN1), in_dtype)
+            P_tmem = T.alloc_tmem([bM, bN1], in_dtype)
+
+            B2_shared = T.alloc_shared((bN2, bN1), in_dtype)
+            D_tmem = T.alloc_tmem([bM, bN2], accum_dtype)
+            mbar2 = T.alloc_barrier(1)
+
+            D_local = T.alloc_fragment((bM, bN2), accum_dtype)
+            D_shared = T.alloc_shared((bM, bN2), out_dtype)
+
+            # Stage 1: SS GEMM — S_tmem = A × B1^T (fp32 accumulator)
```

Ruff (0.15.1)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)